src/ismn/interface.py

```diff
@@ -751,9 +751,9 @@ def get_dataset_ids(self, variable, min_depth=0, max_depth=0.1,
                 * climate_insitu
         """
         lc_cl = ['landcover_2000', 'landcover_2005', 'landcover_2010', 'landcover_insitu', 'climate', 'climate_insitu']
-
-        if max_depth < min_depth:
-            raise ValueError("max_depth can not be less than min_depth")
+        if max_depth:
+            if max_depth < min_depth:
+                raise ValueError("min_depth can not be more than max_depth (default value 0.1)")
```

Comment:
something like  f"min_depth can not be more than max_depth, min_depth is {min_depth} while max_depth is {max_depth}"

Reply:
"min_depth can not be more than max_depth. min_depth: {}, max_depth: {})".format(min_depth, max_depth)